src/map/atcommand.c

```diff
@@ -1714,6 +1714,15 @@ ACMD(bodystyle)
 
 	memset(atcmd_output, '\0', sizeof(atcmd_output));
 
+	if (!((sd->job&MAPID_THIRDMASK) == MAPID_GUILLOTINE_CROSS || (sd->job&MAPID_THIRDMASK) == MAPID_GENETIC
```

Comment:
`sd->job & ` for all here. As well as removing the `if (!(` and adding it as ` == 0` at the end of the statement.